recipes/horton/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "horton" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name | lower }} 
+  version: {{ version }}
+
+source:
+  url: https://github.com/theochem/horton/releases/download/{{ version }}/horton-{{ version }}.tar.gz
+  sha256: 5cd76d53368995dd2fba20903a3bddda1161ffa6ffe9f9eef94189ed0a67cc27
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: {{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - {{ compiler("fortran") }}
+    - cmake
+    - make
+    - git
+  host:
+    - python
+    - setuptools
+    - numpy
+    - scipy
+    - cython
+    - h5py
+    - matplotlib-base
```

Comment:
This is a weird build-time requirement. Are you sure we need it? This section should only contain things needed when `setup.py` is run (e.g. `cython`, or `numpy` if we are linking to the C API). Just wanted to make sure we are not adding things here "just in case".

Reply:
Thanks for catching it! I removed the extraneous dependencies. I think everything else should be required though because we link to the C API for those.